src/database/access.py

```diff
@@ -26,23 +26,21 @@ class AccessDatabase:
     def __enter__(self) -> Self:
         needed_driver: str = access_driver()
         if needed_driver not in pyodbc.drivers():
-            logger.error('Les pilotes ODBC installés sont les suivants :')
+            logger.error('ODBC installed drivers are:')
```

Comment:
```suggestion
            logger.error('Installed ODBC drivers are:')
```
Should also be translated

Reply:
No translations for log strings, don't want to recieve logs in German, Dutch...